frontend/app/routes/_public+/apply+/$id+/review-information.tsx

```diff
@@ -33,58 +34,66 @@ export const meta: MetaFunction<typeof loader> = mergeMeta(({ data }) => {
-  const { id } = await applyFlow.loadState({ request, params });
+  const { id, state } = await applyFlow.loadState({ request, params });
+  const dob = { year: state.dob?.year ?? 2024, month: state.dob?.month ?? 1, day: state.dob?.day ?? 1 };
+  const partnerDob = { year: state.partnerInformation?.year ?? 2024, month: state.partnerInformation?.month ?? 1, day: state.partnerInformation?.day ?? 1 };
 
   const userInfo = {
     firstName: 'John',
     id: '00000000-0000-0000-0000-000000000000',
-    lastName: 'Maverick',
-    phoneNumber: '(754) 628-4776',
-    altPhoneNumber: '(819) 777-1234',
-    preferredLanguage: 'fr',
-    birthday: new Date().toLocaleDateString('en-us', { year: 'numeric', month: 'short', day: 'numeric' }),
-    sin: '123456789',
-    martialStatus: 'Married',
-    email: 'myemail@example.com',
-    communicationPreference: 'email',
+    lastName: state.applicantInformation?.lastName,
+    phoneNumber: state.personalInformation?.phoneNumber,
+    altPhoneNumber: state.personalInformation?.phoneNumberAlt,
+    preferredLanguage: state.communicationPreferences?.preferredLanguage ?? 'en',
+    birthday: new Date(dob.year, dob.month, dob.day).toLocaleDateString('en-us', { year: 'numeric', month: 'short', day: 'numeric' }),
+    sin: state.applicantInformation?.socialInsuranceNumber,
+    martialStatus: state.applicantInformation?.maritalStatus,
+    email: state.communicationPreferences?.email,
+    communicationPreference: state.communicationPreferences?.preferredMethod,
   };
   const spouseInfo = {
     firstName: 'Phil',
     id: '00000000-0000-0000-0000-000000000001',
-    lastName: 'Doe',
+    lastName: state.partnerInformation?.lastName,
     phoneNumber: '(754) 628-4776',
-    birthday: new Date().toLocaleDateString('en-us', { year: 'numeric', month: 'short', day: 'numeric' }),
+    birthday: new Date(partnerDob.year, partnerDob.month, partnerDob.day).toLocaleDateString('en-us', { year: 'numeric', month: 'short', day: 'numeric' }),
     sin: '123456789',
-    consent: true,
-  };
-  const preferredLanguage = {
-    id: 'fr',
-    nameEn: 'French',
-    nameFr: 'Français',
+    consent: state.partnerInformation?.confirm === 'on' ? true : false,
   };
+  const preferredLanguage = (await getLookupService().getPreferredLanguage(userInfo.preferredLanguage)) ?? { id: 'en', nameEn: 'English', nameFr: 'Anglais' };
   const mailingAddressInfo = {
-    address: '44367 Ibrahim Field',
-    city: 'Lake Granvillestead',
-    province: 'PE',
-    postalCode: 'P1L 1C5',
-    country: 'CAN',
+    address: state.personalInformation?.mailingAddress ?? 'Unknown',
+    appartment: state.personalInformation?.mailingApartment ?? 'Unknown',
+    city: state.personalInformation?.mailingCity ?? 'Unknown',
+    province: state.personalInformation?.mailingProvince ?? 'Unknown',
+    postalCode: state.personalInformation?.mailingPostalCode ?? 'Unknown',
+    country: state.personalInformation?.mailingCountry ?? 'Unknown',
```

Comment:
I dislike having all these default values for what I assume are mandatory fields for the application. I wonder if we should have a different validation schema for the entire application rather than re-using the flow state schema, which has every page in the flow as optional.

cc @gregory-j-baker @sebastien-comeau

Reply:
I think the validation should be moved into their respective pages, and exported from there.

The final step that will store the data via an API call should then validate the data as a whole before submitting. This can be done by simply setting a `validated: true` flag in the data for a specific step, or by re-running the data through the exported validators.

As for the defaults here, I just kind of assumed that this is unfinished placeholder code (is it not?).

Reply:
Sorry.. to be clear, my comment about moving the validation into each respective .tsx file is not for this PR.. so don't worry too much about that.

Reply:
Resolving for future PR